clang/docs/ReleaseNotes.rst

```diff
@@ -1045,6 +1045,9 @@ X86 Support
 Arm and AArch64 Support
 ^^^^^^^^^^^^^^^^^^^^^^^
 
+- Support for SVE2.1 and SME2.1 using the Arm C Language Extensions (ACLE) is
```

Comment:
I am not sure if "using" is the best work, maybe "according to".
I looked on how Momchil did for BitInt:
https://github.com/llvm/llvm-project/pull/101521/files
Maybe :
lmplementation of SVE2.1 and SME2.1 in accordance with the Arm C Language Extensions (ACLE) is now available

Reply:
Done.